integration/conftest.py

```diff
@@ -455,6 +307,21 @@ def bob(reactor, temp_dir, introducer_furl, flog_gatherer, storage_nodes, reques
 @pytest.mark.skipif(sys.platform.startswith('win'),
                     'Tor tests are unstable on Windows')
 def chutney(reactor, temp_dir: str) -> tuple[str, dict[str, str]]:
+    """
+    Instantiate the "networks/hs-v3" Chutney configuration for a local
+    Tor network.
```

Comment:
It looks like the network configuration choice isn't actually made here.

Reply:
thanks .. also some other problems; re-worked the comments.